tools/rosbash/scripts/rosrun

```diff
@@ -66,15 +68,18 @@ if [[ ! $2 == */* ]]; then
     _perm="/111"
   fi
   debug "Searching for $2 with permissions $_perm"
-  exepathlist=(`find -L $catkin_package_libexec_dir $pkgdir -name $2 -type f  -perm $_perm ! -regex ".*$pkgdir\/build\/.*" | uniq`)
+  exepathlist="`find -L "${catkin_package_libexec_dirs[@]}" "$pkgdir" -name "$2" -type f  -perm "$_perm" ! -regex ".*$pkgdir\/build\/.*" | uniq`"
```

Comment:
Ah, I think there is a slight misconception here how arrays are expanded.

Suppose you have the array `array=("aap noot mies" "wim zus jet")`.
Then `"${array[@]}` expands to `"aap noot mies" "wim zus jet"` (one word per array element) and not to `"aap noot mies wim zus jet"`.

That's the entire reason I switched `$catkin_package_libexec_dirs` to an array.